docs/pages/sidenavigation.js

```diff
@@ -0,0 +1,638 @@
+// @flow strict
+import { type Node } from 'react';
+import { SideNavigation, SlimBanner } from 'gestalt';
+import MainSection from '../components/MainSection.js';
+import PageHeader from '../components/PageHeader.js';
+import Page from '../components/Page.js';
+import GeneratedPropTable from '../components/GeneratedPropTable.js';
+import { multipledocgen, type DocGen } from '../components/docgen.js';
+import AccessibilitySection from '../components/AccessibilitySection.js';
+
+export default function SideNavigationPage({
+  generatedDocGen,
+}: {|
+  generatedDocGen: {| [string]: DocGen |},
+|}): Node {
+  return (
+    <Page title={generatedDocGen.SideNavigation?.displayName ?? ''}>
+      <PageHeader
+        name={generatedDocGen.SideNavigation?.displayName}
+        description={generatedDocGen.SideNavigation?.description}
+        shadedCodeExample
+        defaultCode={`
+<SideNavigation accessibilityLabel="Main example">
+  <SideNavigation.Item
+    href="#"
+    onSelect={({ event }) => event.preventDefault}
+    icon="bell"
+    item={{ label: 'Notifications', value: 'notifications' }}
+    counter={{ number: '20', accessibilityLabel: 'You have 20 notifications in your inbox' }}
+    notificationAccessibilityLabel="New notifications"
+  />
+  <SideNavigation.Item
+    href="#"
+    onSelect={({ event }) => event.preventDefault}
+    icon="speech"
+    item={{ label: 'Messages', value: 'messages' }}
+    counter={{ number: '10', accessibilityLabel: 'You have 10 messages in your inbox' }}
+    notificationAccessibilityLabel="New messages"
+  />
+  <SideNavigation.Item
+    href="#"
+    onSelect={({ event }) => event.preventDefault}
+    icon="cog"
+    item={{ label: 'Settings', value: 'settings' }}
+  />
+  <SideNavigation.Item
+    href="#"
+    onSelect={({ event }) => event.preventDefault}
+    icon="lock"
+    item={{ label: 'Business Access', value: 'business' }}
+  />
+  <SideNavigation.Item
+    href="#"
+    onSelect={({ event }) => event.preventDefault}
+    icon="add-layout"
+    item={{ label: 'Tune your home feed', value: 'tune_homefeed' }}
+    badge={{ text: 'New', type: 'info' }}
+  />
+</SideNavigation>
+        `}
+        slimBanner={
+          <SlimBanner
+            type="warning"
+            iconAccessibilityLabel="Warning"
+            message="This component is on alpha phase, still under developoment. The component will support three nested levels and accessibility keyboard navigation. The component will change behavior and the API might also change in future component version releases."
```

Comment:
"This component is in the alpha phase and is still under development."

Reply:
DONE

Reply:
DONE